scope/fritz.py

```diff
@@ -90,7 +91,19 @@ def api(
         kwargs["json"] = data
     elif method == "GET":
         kwargs["params"] = data
-    response = requests.request(**kwargs)
+
+    for attempt in range(max_attempts):
```

Comment:
There should be some kind of sleep here?

Reply:
Probably that sleep should be set as a function argument. Defaulting to 5 seconds?